### PR TITLE
Update Understand-determine-basal.md

### PR DESCRIPTION
**Pull request by parenthetic**
- added description of short_avgdelta and long_avgdelta
- put json keys in code blocks (only for Summary of Inputs -- haven't gotten to Summary of Outputs section (yet))

### CODE REVIEW

File: docs/docs/walkthrough/phase-3/Understand-determine-basal.md [hunk 1]
```diff
@@ -10,42 +10,43 @@ The determine-basal algorithm requires a number of inputs, which are passed in J
...
 ```
 {"carbs":0,"boluses":0}
-{"delta":-2,"glucose":110,"avgdelta":-2.5}
+{"delta":5,"glucose":161,"short_avgdelta":4.5,"long_avgdelta":3.92}
 {"duration":0,"rate":0,"temp":"absolute"}
 {"iob":0,"activity":0,"bolussnooze":0,"basaliob":0,"netbasalinsulin":0,"hightempinsulin":0,"time":"2017-03-17T00:34:51.000Z"}
 {"carbs_hr":28,"max_iob":1,"dia":3,"type":"current","current_basal":1.1,"max_daily_basal":1.3,"max_basal":3,"max_bg":120,"min_bg":115,"carbratio":10,"sens":40}
 ```
 
 * meal.json = `{"carbs":0,"boluses":0}`
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-3/Understand-determine-basal.md]: At some point we should add mealCOB here too.

File: docs/docs/walkthrough/phase-3/Understand-determine-basal.md [hunk 2]
```diff
@@ -10,42 +10,43 @@ The determine-basal algorithm requires a number of inputs, which are passed in J
...
-  * carbs = # of carbs consumed
-  * boluses = amount of bolus insulin delivered
+  * `carbs` = # of carbs consumed
+  * `boluses` = amount of bolus insulin delivered
   
   Those data come from what you entered into your pump or Nightscout web app. If provided, allows determine-basal to decide when it is appropriate to enable Meal Assist.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-3/Understand-determine-basal.md]: And update this to Advanced Meal Assist when we add mealCOB.

Comment (parenthetic) [inline docs/docs/walkthrough/phase-3/Understand-determine-basal.md]: Totally agree.

Still running in low glucose suspend mode without AMA turned and working through the code.

Next up for me is the IOB calcs. I've replicated the OpenAPS calcs here (https://github.com/parenthetic/Replicate-OpenAPS-IOB-calcs) and produced some graphs, which I hope to add into the docs. But likely won't get to them until tomorrow. Any thoughts on those are certainly welcome...

**Comment by danamlewis** [comment]
@scottleibrand is that a blocker for merging this, or just "nice to do" in future?

**Pull request merged**